privacyidea/lib/tokenclass.py

```diff
@@ -1566,22 +1566,17 @@ def get_setting_type(key):
         return ""
 
     @classmethod
-    def get_default_settings(cls, params, logged_in_user=None,
-                             policy_object=None, client_ip=None):
+    def get_default_settings(cls, g, params):
```

Comment:
Please add changes in the tokenclass interface to `READ_BEFORE_UPDATE` because it breaks other token classes.

Reply:
done in 36a31028